torch/onnx/utils.py

```diff
@@ -471,6 +473,21 @@ def _get_example_outputs(model, args):
     return example_outputs
 
 
+_qtype_vtype_map = {torch.quint8: torch.uint8, torch.qint8: torch.int8, torch.qint32: torch.int32, torch.quint4x2: torch.int8}
+
+
+def unpack_quantized_tensor(tensor):
```

Comment:
The input can be either tensor or arithmetic values. For example, `test_arithmetic_prim_bool` the input y, z, t. Rename `tensor` to `value` to avoid confusion.